.circleci/config.yml

```diff
@@ -125,12 +124,29 @@ jobs:
           name: Push Docker image to Quay
           command: |
             docker login -u $DOCKER_USER -p $DOCKER_PASS quay.io
-            docker tag quay.io/plotly/image-exporter:$CIRCLE_SHA1 quay.io/plotly/image-exporter:$CIRCLE_BRANCH
             docker push quay.io/plotly/image-exporter:$CIRCLE_SHA1
-            docker push quay.io/plotly/image-exporter:$CIRCLE_BRANCH
-            if [[ ! -z "${CIRCLE_TAG}" ]]; then
-              docker tag quay.io/plotly/image-exporter:$CIRCLE_SHA1 quay.io/plotly/image-exporter:$CIRCLE_TAG
-              docker push quay.io/plotly/image-exporter:$CIRCLE_TAG
+
+            # Is this a "version" commit? e.g. from `npm version`
+            # inspired by: https://discuss.circleci.com/t/git-tag-deploys-in-2-0/9493/4
+            if git log -1 --pretty=%B | grep "^[0-9]\+\.[0-9]\+\.[0-9]\+$"; then
```

Comment:
This regex wouldn't match anything starting with a `v`. I think it should, shouldn't it?